library/declare.mli

```diff
@@ -87,7 +87,9 @@ val exists_name : Id.t -> bool
 
 
 
-(** Global universe names and constraints *)
+(** Global universe contexts, names and constraints *)
 
+val declare_universe_context : polymorphic -> Univ.universe_context_set -> unit
```

Comment:
I'll leave a general labelization of ~polymorphic for later (it should rather be done everywhere at once)